packages/front-end/package.json

```diff
@@ -96,6 +96,8 @@
     "@swc/cli": "^0.1.57",
     "@swc/core": "^1.3.4",
     "@swc/jest": "^0.2.23",
+    "@testing-library/dom": "^10.4.0",
```

Comment:
I don't see you importing this anywhere.  Was it an alternative to jest-dom that you were trying out, or is it needed for jest-dom to work correctly?